app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -86,8 +86,13 @@ EOF
 elif [ -f /etc/debian_version ]; then
   <%= save_to_file('/etc/apt/sources.list.d/foreman_registration.list', @repo) %>
 <% if @repo_gpg_key_url.present? -%>
-  apt-get -y install ca-certificates gpg
-  curl --silent --show-error --output /etc/apt/trusted.gpg.d/client.asc <%= shell_escape @repo_gpg_key_url %>
+  $PKG_MANAGER_INSTALL ca-certificates curl gnupg
+<% # "apt 1.2.35" on Ubuntu 16.04 does not support storing GPG public keys in "/etc/apt/trusted.gpg.d/" in ASCII format -%>
```

Comment:
ERB has `<# #>` for comments

Reply:
I think it's `<%# %>`: https://github.com/ruby/erb?tab=readme-ov-file#recognized-tags

Reply:
Thanks @ofedoren 

Applied.